src/Badger/Bundle/UserBundle/Entity/User.php

```diff
@@ -37,6 +37,9 @@ class User extends BaseUser implements UserInterface, TaggableInterface
     /** @var int */
     protected $nuts;
 
+    /** @var \DateTime */
+    protected $date_registered;
```

Comment:
we already have snake_case variable in this file, so I wanted to be consistent :(